pkg/sql/grant_revoke.go

```diff
@@ -176,16 +176,15 @@ type changeNonDescriptorBackedPrivilegesNode struct {
 // and expects to see its own writes.
 func (n *changeDescriptorBackedPrivilegesNode) ReadingOwnWrites() {}
 
-func (n *changePrivilegesNode) preChangePrivilegesValidation(params runParams) error {
-	ctx := params.ctx
-	p := params.p
-
-	if err := p.validateRoles(ctx, n.grantees, true /* isPublicValid */); err != nil {
+func (p *planner) preChangePrivilegesValidation(
```

Comment:
Why did this move to the planner?

Reply:
so that i could call this function from `alter_default_privileges.go`, which doesn't have a `changePrivilegesNode` to call this with.